tests/jballerina-debugger-integration-test/src/main/java/org/ballerinalang/debugger/test/utils/BallerinaTestDebugPoint.java

```diff
@@ -28,12 +28,19 @@
 public class BallerinaTestDebugPoint {
     String filePath;
     long line;
+    String condition;
 
     public BallerinaTestDebugPoint(String filePath, long line) {
         this.filePath = filePath;
         this.line = line;
     }
 
+    public BallerinaTestDebugPoint(String filePath, long line, String condition) {
+        this.filePath = filePath;
+        this.line = line;
+        this.condition = condition;
+    }
```

Comment:
Shall we reuse this constructor in the old constructor?

Reply:
Fixed